kahuna/public/js/util/async.js

```diff
@@ -56,7 +56,7 @@ const queue = new PQueue({ concurrency: 10 });
 async.factory("apiPoll", [
   () => {
     const wait = () => new Promise(resolve => {
-      setTimeout(() => resolve(), 4000);
+      setTimeout(() => resolve(), 500);
```

Comment:
Just to double check – does this mean (4000/500)/s more polls per image in batch until timeout (my read)? Or does this only happen once? Probably this isn't too problematic: my understanding is that we only poll for a limited number of images at once, and our API is pretty robust, but I want to check that a) my understanding is correct and that b) we've thought about the impact on the service.

Reply:
Yes it does, this was the old poll rate, and now we know that polling isn't the weak link, we can go back to it.